README.md

```diff
@@ -1,2 +1,30 @@
 # hello-world
 Baby's First Repository
+
+New to coding, trying to teach myself as much as I can, but I do happen to have a really amazing tutor. 
+Hobbies include; learning to code, hanging with my favorite person, being a mom, running, writing, cooking, baking, music, tattoos, beer, whiskey, movies, talking smack, tarot, eating tacos, and smashing the patriarchy whenever possible. 
```

Comment:
I would use a colon `:` here instead of a semi-colon `;`.

Because this is a markdown file (which you can tell by the .md file extension), you can also use all kinds of cool, fancy formatting tricks (italics, bold, lists, embedded images, etc). Here is a [great guide to learning markdown syntax](https://guides.github.com/features/mastering-markdown/) that you should take a look at.

I'll approve the PR once you add some fanciness ;)

Reply:
You are right about the colon. Dammit. I hate it when you're right. 

I'll work on the fancy.

Reply:
Notice - I did change what I could to use the markdown syntax you suggested.